MIGRATION_GUIDE.md

```diff
@@ -1,5 +1,88 @@
 # Migration Guide
 
+## Upgrading from v0.49.0 → v0.50.0
+
+There are deprecations in this update. Please ensure you read this guide thoroughly and prepare your potential
+automated workflows before upgrading.
+
+### Deprecations
+
+- [Auth0 Pages](#auth0-pages)
```

Comment:
IMO makes more sense to split out into two sections for tenant and global client.